ledger/participant-state/kvutils/src/main/protobuf/daml_kvutils.proto

```diff
@@ -494,3 +442,71 @@ message LedgerDumpEntry {
   // The resulting output state.
   repeated StatePair output_state = 5;
 }
+
+// Errors
+
+// The transaction relied on contracts or keys being active that were no longer active
+message Inconsistent {
+  string details = 1;
+}
+
+// The transaction has been disputed
```

Comment:
Add comment that disputed error happens when the DAML model conformance or authorization fails.

Reply:
Done